linkis-engineconn-plugins/spark/pom.xml

```diff
@@ -27,6 +27,7 @@
   <artifactId>linkis-engineplugin-spark</artifactId>
   <properties>
     <spark.version>2.4.3</spark.version>
+    <hadoop.version>2.7.2</hadoop.version>
```

Comment:
this will have multi hadoop client when the spark ec start.
the ec will include the linkis-commons,  with include the hadoop client